rkt/config/directories.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"encoding/json"
+)
+
+type configurableDirectoriesV1 struct {
+	Data string `json:"data"`
+}
+
+func init() {
+	addParser("directories", "v1", &configurableDirectoriesV1{})
+	// Look in 'directories' subdir for configs of type directories
+	registerSubDir("directories", []string{"directories"})
```

Comment:
I would give shorter name for configuration subdirectory. Also, as a convention, we suffix those dirs with `.d`. So something like `dirs.d` would be better IMHO.